extensions/etcdsnapshot/etcdsnapshot.go

```diff
@@ -165,41 +122,58 @@ func CreateRKE2K3SSnapshot(client *rancher.Client, clusterName string) error {
 	}
 
 	logrus.Infof("Creating snapshot...")
-	_, err = client.Steve.SteveType(clusters.ProvisioningSteveResourceType).Update(clusterSteveObject, clusterObject)
+	updatedCluster, err := client.Steve.SteveType(clusters.ProvisioningSteveResourceType).Update(clusterSteveObject, clusterObject)
 	if err != nil {
-		return err
+		return nil, err
+	}
+
+	updateTimestamp := updatedCluster.ManagedFields[len(updatedCluster.ManagedFields)-1].Time.Time
```

Comment:
Calling this out. The linter caught an `ineffassign` that needs to be addressed.